File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <iostream>
using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;

    int sum = a + b + c;

    if (sum != 180) {
        cout << "Error" << endl;
    } else if (a == 60 && b == 60 && c == 60) {
        cout << "Equilateral" << endl;
    } else if (a == b || b == c || a == c) {
        cout << "Isosceles" << endl;
    } else {
        cout << "Scalene" << endl;
    }

    return 0;
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
import sys

a = int(sys.stdin.readline())
b = int(sys.stdin.readline())
c = int(sys.stdin.readline())

if a + b + c != 180:
    print("Error")
elif a == b == c == 60:
    print("Equilateral")
elif a == b or b == c or a == c:
    print("Isosceles")
else:
    print("Scalene")

